components/match2/wikis/rainbow_six/match1 to match2/legacy_bracket_match_summary.lua

```diff
@@ -0,0 +1,68 @@
+---
+-- @Liquipedia
+-- wiki=rainbowsix
+-- page=Module:LegacyBracketMatchSummary
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+--[[
+	This module is used in the process convert Legacy match1 into match2.
+	It converts a few fields to new format, and jsonify the args.
+	It is invoked by Template:BracketMatchSummary.
+]]
+
+local Json = require('Module:Json')
+local String = require('Module:String')
+local Class = require('Module:Class')
+
+local LegacyBracketMatchSummary = {}
+
+function LegacyBracketMatchSummary.convert(args)
+	if not args then
+		return
+	end
+	local index = 1
+	while not String.isEmpty(args['map' .. index]) or not String.isEmpty(args['map' .. index .. 'win']) do
+		local prefix = 'map' .. index
+		local score1
+		local score2
+		if not String.isEmpty(args[prefix..'score']) then
+			score1 = mw.text.split(args[prefix..'score'], '-')[1]
+			score2 = mw.text.split(args[prefix..'score'], '-')[2]
+		end
+		if not score1 and not score2 and args[prefix ..'win'] ~= 'skip' then
+			score1 = tonumber(args[prefix ..'win']) == 1 and 'W' or 'L'
+			score2 = tonumber(args[prefix ..'win']) == 2 and 'W' or 'L'
+		end
+
+		args['map' .. index .. 'score1'] = mw.text.trim(score1 or '')
+		args['map' .. index .. 'score2'] = mw.text.trim(score2 or '')
+		args['map' .. index .. 'score'] = nil
+		args['map' .. index .. 'finished'] = (args[prefix ..'win'] == 'skip' and 'skip') or
+											 (not String.isEmpty(args[prefix ..'win']) and 'true') or 'false'
```

Comment:
indents look weird (at least on tablet)

Reply:
![bild](https://user-images.githubusercontent.com/3426850/147395400-ab9a156f-b792-4894-8bb1-16bb80867c79.png)
Looks fine to me in IDE (tab=4 space)